build/build.sh

```diff
@@ -33,53 +33,19 @@ pandoc --verbose \
   --bibliography=$BIBLIOGRAPHY_PATH \
   --csl=$CSL_PATH \
   --metadata link-citations=true \
-  --mathjax \
   --include-after-body=build/themes/default.html \
-  --include-after-body=build/plugins/analytics.html \
```

Comment:
Was the intention to remove the analytics and require the user to put it back in the build command? There's not much harm keeping it enabled by default.